kivy/network/urlrequest.py

```diff
@@ -75,7 +75,6 @@ def bug_posted(req, result):
 try:
     import ssl
 
-    HTTPSConnection = None
```

Comment:
Later into the code, `HTTPSConnection` is checked to not being `None` as we support this case.